sdk/keyvault/azure-keyvault-administration/azure/keyvault/administration/_models.py

```diff
@@ -140,13 +140,15 @@ def __repr__(self) -> str:
     def _from_generated(cls, definition: RoleDefinition) -> "KeyVaultRoleDefinition":
         # pylint:disable=protected-access
         return cls(
-            assignable_scopes=definition.assignable_scopes,
-            description=definition.description,
+            assignable_scopes=definition.properties.assignable_scopes if definition.properties else [],
+            description=definition.properties.description if definition.properties else "",
```

Comment:
qq: Are we changing the defaults here or were they always empty list or empty string?

Reply:
Good point; after double-checking, all of these properties were previously `Optional` in the generated type, and still are after the restructuring. That's consistent with what KV tends to do, which is why pretty much all of our model properties are `Optional`... I should update these defaults to `None` and update our type hints